src/librustc_resolve/macros.rs

```diff
@@ -385,10 +385,9 @@ impl<'a> Resolver<'a> {
         if let Some(suggestion) = find_best_match_for_name(self.macro_names.iter(), name, None) {
             if suggestion != name {
                 err.help(&format!("did you mean `{}!`?", suggestion));
-            } else {
-                err.help(&format!("have you added the `#[macro_use]` on the module/import?"));
```

Comment:
I think we should keep this here, but it should just say "have you added the `#[macro_use]` on the module?".